src/Symfony/Bundle/SecurityBundle/Resources/config/security_authenticator.xml

```diff
@@ -60,10 +60,8 @@
         </service>
 
         <service id="security.listener.session"
-                 class="Symfony\Component\Security\Http\EventListener\SessionStrategyListener"
-                 abstract="true">
```

Comment:
this `abstract="true"` must be kept (the service itself should not be used, only the onces created based on this in the SecurityExtension)